home/.bash_aliases

```diff
@@ -1,17 +1,66 @@
 #!/usr/bin/env bash
 
+# Easier navigation: .., ..., ...., ....., ~ and -
 alias cd..="cd .."
 alias ..="cd .."
-alias ll="ls -lhA"
+alias ...="cd ../.."
+alias ....="cd ../../.."
+alias .....="cd ../../../.."
+
+# Shortcuts
+alias d="cd ~/Documents/Dropbox"
+alias dl="cd ~/Downloads"
+alias dt="cd ~/Desktop"
+alias p="cd ~/Projects"
+
+# JFV: Not sure what to do about colors, add this as a placeholder
```

Comment:
No need to add initials. Each line in a file is tracked by git and your user is associated with that change. Git has a 'blame' report that shows who wrote or modified every line and when the change happened. You can jump back to old commits and get a sense of the history from there. 

So this is an old pattern. Embrace git. No need to leave your name in the files.